### ISSUE TITLE
🔤 Plain language copy updates to the `SkillBrowser`

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#8423]
## 🔤 'SkillBrowser' copy issue 

Three copy updates to the skills browser. See acceptance criteria.

## 🎨 Design Files

> [Skills Library (Skill picker dialogue)](https://www.figma.com/file/5LcwA4AmOM8ebdJKuiPs8g/Skills-library?type=design&node-id=155-5147&mode=design&t=1E5MWAvwf6py5aYJ-0)
> [Talent Search](https://www.figma.com/file/9V8kBIWxOXRaN9dkcVCU1o/Manager---Talent-search?type=design&node-id=204-1593&mode=design&t=1E5MWAvwf6py5aYJ-0)
> [Job Application V6 (Select an experience) ](https://www.figma.com/file/BXYVsu1fo9LS2IySJDUINY/Candidates---Job-application---Version-1-to-6?type=design&node-id=547-49171&mode=design&t=1E5MWAvwf6py5aYJ-0)
> [Pools Record of Decision (Add Essential and Asset Skill modals)](https://www.figma.com/file/em5ArPr044YaIe1Vb1zCZz/Pools---Record-of-Decision?type=design&node-id=3513-11411&mode=design&t=1E5MWAvwf6py5aYJ-0)

### 🧑‍🎨 Designers

@RM-1978 

## 📖 Storybook component

> [SkillBrowser](https://61e099a1bb1465003a73d3ce-sdnauqrpyl.chromatic.com/iframe.html?id=components-skill-browser-inline--default&viewMode=story)

### 👩‍💻  Developer 

Eric Sizer (@esizer) 

## 🕵️ Details

Three copy instances where users: 

- "Select an option" null state label being read as: "disabled selection option 1." AT users cannot easily determine that they are being presented with filtering options.  
- Bracketed numbers at the end of each option are confusing  without context, and conflict with the selection option numbering read and announced by screen readers.
- Once you select this option, all skills from every family appear. The skills are not presented by family.

## ✅ Acceptance Criteria

- [ ] Update the null state with the component context: "Select a skill family"
- [ ] Update "All skill families" to read " All skills." 
- [ ] Add the word "skills" to the remaining menu options beside the count (i.e., 10 skills).

**Comment by gobyrne** [comment]
@Jerryescandon Update this one too please :)
